ddtrace/profiling/bootstrap/sitecustomize.py

```diff
@@ -27,5 +27,12 @@ def start_profiler():
         "If you believe this is an error or need assistance, please report it at "
         "https://github.com/DataDog/dd-trace-py/issues"
     )
+elif platform.system() == "Windows":
+    LOG.error(
+        "The Datadog Profiler is not supported on Windows. "
+        "To use the profiler, please use a 64-bit Linux or macOS system. "
+        "If you believe this is an error or need assistance, please report it at "
```

Comment:
```suggestion
        "If you need assistance related to Windows support for the Profiler, please open a ticket at "
```
"if you believe this is an error" could be read as passive-aggressive